### PR TITLE
Feature: refactor process environment variables

### PR DESCRIPTION
**Pull request by oyershov**
- prevent crashes on missing variables
- put default variables on missing from the config

### CODE REVIEW

File: src/api/config.ts [hunk 1]
```diff
@@ -1,79 +1,66 @@
...
-import { Config } from './types';
-
 export const defaultConfig: Config = {
     api: {
         authUrl: '',
         tradeUrl: '',
         applogicUrl: '',
         rangerUrl: '',
-        finexUrl: '',
+        finexUrl: ''
```
Comment (akhlopiachyi) [inline src/api/config.ts:7]: add comma

File: src/api/config.ts [hunk 2]
```diff
@@ -1,79 +1,66 @@
...
+    captchaLogin: false,
+    gaTrackerKey: '',
+    minutesUntilAutoLogout: '5',
+    msAlertDisplayTime: '5000',
+    sessionCheckInterval: '15000',
+    balancesFetchInterval: '3000',
+    passwordEntropyStep: '14',
+    storage: {
+        defaultStorageLimit: '50',
+        orderBookSideLimit: '25'
```
Comment (akhlopiachyi) [inline src/api/config.ts:25]: add comma

File: src/api/config.ts [hunk 3]
```diff
@@ -1,79 +1,66 @@
...
+    languages: ['en', 'ru'],
     kycSteps: [
         'email',
         'phone',
         'profile',
         'document',
-        'address',
-    ],
-    captchaLogin: false,
+        'address'
```
Comment (akhlopiachyi) [inline src/api/config.ts:33]: add comma

File: src/helpers/jest.ts [hunk 4]
```diff
@@ -15,21 +15,32 @@ const mockConfig: Config = {
...
+    captchaLogin: false,
+    gaTrackerKey: '',
+    minutesUntilAutoLogout: '5',
+    msAlertDisplayTime: '5000',
+    sessionCheckInterval: '15000',
+    balancesFetchInterval: '3000',
+    passwordEntropyStep: '14',
+    storage: {
+        defaultStorageLimit: '50',
+        orderBookSideLimit: '25'
```
Comment (akhlopiachyi) [inline src/helpers/jest.ts:34]: add comma

File: src/helpers/jest.ts [hunk 5]
```diff
@@ -15,21 +15,32 @@ const mockConfig: Config = {
...
+    },
+    languages: ['en', 'ru'],
+    kycSteps: [
+        'email',
+        'phone',
+        'profile',
+        'document',
+        'address'
```
Comment (akhlopiachyi) [inline src/helpers/jest.ts:42]: add comma

File: src/modules/public/ranger/sagas/rangerSaga.test.ts [hunk 6]
```diff
@@ -48,8 +48,7 @@ describe('Ranger module', () => {
                 applogicUrl: '',
                 rangerUrl: `ws://localhost:${echoServerPort}`,
                 finexUrl: '',
-            },
-            rangerReconnectPeriod: '0.1',
+            }
```
Comment (akhlopiachyi) [inline src/modules/public/ranger/sagas/rangerSaga.test.ts:51]: add comma

**Review by josadcha — APPROVED** [review]

**Review by akhlopiachyi — APPROVED** [review]

**Pull request merged**